lib/log_parser.rb

```diff
@@ -9,4 +11,20 @@ def read_first_line
     firstLine = File.open(@file_path, &:readline)
   end
 
+  def count_lines
+    file = File.open(@file_path)
+    line_count = file.readlines.count
+    file.close
+    line_count
+  end
+
+  def logJSON
```

Comment:
Por convenção, no ruby funções são declaradas utilizando o formato `snake_case`

```suggestion
  def log_JSON
```